src/components/OSCALMarkdownProse.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+
+export default function OSCALMarkdownProse(props) {
+  return <ReactMarkdown>{props.text}</ReactMarkdown>;
```

Comment:
You can also implement this with a single tag
`<ReactMarkdown children = {props.text}/>
It is something you can consider, it can make the code cleaner

Reply:
I tried that and was getting a linting error